src/DotVVM.Framework/Hosting/IDotvvmRequestContext.cs

```diff
@@ -111,5 +111,7 @@ public interface IDotvvmRequestContext
         string? ResultIdFragment { get; set; }
 
         IServiceProvider Services { get; }
+        object? CommandResult { get; set; }
```

Comment:
Agree that this property does not have to be there. But do you have some idea how to provide an interface to modify the result of command or static command in OnCommandExecutedAsync and etc.?

Reply:
Just add an (`ref`) parameter to the `OnCommandExecutedAsync` method. Since most people use the base class, we can add a second overload and prevent a breaking change.